src/rpc/client/ts/webpack.config.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2021 Concurrent Technologies Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path')
+const DtsBundleWebpack = require('dts-bundle-webpack')
```

Comment:
This package provides no special benefits. The `index.d.ts` is already auto generated to the `out` directory where it needs to be during the `webpack` building. Going to remove this package but I like the copy plugin in one. Except the fact that it doesn't copy the top directory of `omega-edit-grpc-server`. So that will be fixed in some scripting but simplifies our process a good bit more.